src/test/java/org/opentripplanner/netex/NetexLoaderSmokeTest.java

```diff
@@ -83,6 +86,15 @@ private void assertAgencies(Collection<Agency> agencies) {
         assertEquals(1, agencies.size());
     }
 
+    private void assertOperators(Collection<Operator> operators) {
+        Operator o = list(operators).get(0);
+        assertEquals("RUT:Operator:130c", o.getId().getId());
+        assertEquals("Ruter", o.getName());
+        assertNull( o.getUrl());
+        assertNull(o.getPhone());
+        assertEquals(1, operators.size());
```

Comment:
I find the body more readable if the size is asserted first, because the Collection is not ordered, so the get(0) operation does not appear deterministic until you know its length is one.